packages/swap/src/quoting/authenticate.ts

```diff
@@ -101,14 +99,39 @@ const authenticate = async (socket: QuotingSocket, next: Next) => {
       quotedAssets: auth.quoted_assets,
       clientVersion: auth.client_version,
       beta: marketMaker.beta,
-      mevFactors: marketMaker.mevFactors.reduce(
-        (acc, mev) => {
-          if (env.QUOTER_USE_MEV_FACTOR) {
-            acc[toLowerCase(mev.side)][mev.asset] = mev.factor * (mev.side === 'BUY' ? 1 : -1);
+      mevFactors: marketMaker.factors.reduce(
+        (acc, factor) => {
+          if (factor.type === 'MEV' && env.QUOTER_USE_MEV_FACTOR) {
+            if (factor.side === null) {
+              logger.error('factor side is null', { factor });
+              return acc;
+            }
+            acc[toLowerCase(factor.side)][factor.asset] =
+              factor.factor * (factor.side === 'BUY' ? 1 : -1);
+          }
+          return acc;
+        },
+        { buy: createInternalAssetMap(0), sell: createInternalAssetMap(0) },
+      ),
+      replenishmentFactors: marketMaker.factors.reduce(
+        (acc, factor) => {
+          if (factor.type === 'REPLENISHMENT') {
```

Comment:
we could perhaps filter them outside of the reducer, for both the `replenishmentFactors` and `mevFactors`